test/Serialization/Fixtures/SerializableObjectUsingTrait.php

```diff
@@ -2,7 +2,7 @@
 
 namespace BroadwaySerialization\Test\Serialization\Fixtures;
 
-use Broadway\Serializer\SerializableInterface;
+use Broadway\Serializer\Serializable as SerializableInterface;
```

Comment:
True, then I agree we should rename it to `AutoSerializable`. I can do it later today